src/lib/cli/pager.js

```diff
@@ -7,6 +7,18 @@ import { spawn } from 'child_process';
 
 let proc;
 export default function pager() {
+	let less;
+	switch ( process.platform ) {
+		case 'win32':
+			// PROGRA~1 is the short name for Program Files
+			// we're using it here to avoid the space which is broken by .split( ' ' )
+			less = 'C:\\PROGRA~1\\Git\\usr\\bin\\less.exe -FRX';
```

Comment:
@mjangda this works in cmd.exe and Git bash for me.

Reply:
I guess there will be the edge case of users who choose not to install stuff in `Program Files`. 

How easy would it be to fallback to `console.log` in cases where we can't find an appropriate pager lib? Is it worth factoring in?

Reply:
I think it would be easier to bundle `less.exe` and reference it relative to the CLI.

Reply:
I thought maybe https://github.com/Automattic/vip/pull/139/commits/157f9e634b2a1537cd914431aead1582daf02436 would work, but it still triggers the uncaught error handler :(

Reply:
My 2 cents as I am sure you already discussed this, but I think if we want to support Windows properly, we should probably either check for `less.exe` existence on target systems by using something like `where`, or at least give clients an option to specify a PATH to the exe? Or is there a more reliable way to understand if the binary is present on Windows (perhaps checking `%path`?).
If we'd chose to bundle it instead, which version?
Probably not a huge deal, depending on how badly we want to support Windows.

Reply:
I dealt with this in https://github.com/Automattic/vip/pull/139/commits/887ee33f668c15de5b96110fcf11f85245bff73f by falling back to sending output directly to `process.stdout` if the `less` or `less.exe` binary doesn't exist (or any other problems we run into trying to exec that binary).

If we want to check whether the binary exists first, I think we'd want a pure Node implementation of `where`/`which`/`type` because otherwise it's going to get pretty complicated. We need to `exec()` to run the program that checks for the existence of another program, get the output from that, and then decide where we're going to output. It also adds another "if Windows" conditional because the program to check if another program exists is different.

We do always check `process.env.PAGER`, so if you've got less installed in a non-standard place, you could specify it.

Git is required to run `npm install`, which is currently required to install our CLI, so I think it's going to be rare that `less.exe` is missing, but if it is, there's a graceful fallback.